.github/workflows/build-pull-request.yml

```diff
@@ -78,37 +78,25 @@ jobs:
         name: reports
         path: Artifacts
 
-    - name: Download Base API Spec
-      id: base-spec
-      uses: TizenAPI/tizenfx-build-actions/s3-download-public@master
-      with:
-        region: ap-northeast-2
-        bucket: tizenfx-api-specs
-        key: ${{ github.event.pull_request.base.sha }}.json
-        path: Artifacts/base-api.json
-
-    - uses: actions/checkout@v3
-      if: ${{ steps.base-spec.outputs.code != 200 }}
+    - name: PR event trigger
```

Comment:
Maybe it is better to name it something like "Checkout to base spec"? "Checkout to base  API spec"?

Reply:
OK `Checkout to base spec` is better

Reply:
Thanks!